scripts/memory-threshold-list.yaml

```diff
@@ -0,0 +1,36 @@
+- scenarios:
+    - sample.matter.lock.release
+    - sample.matter.light_bulb.release
+  platforms:
+    - nrf52840dk_nrf52840
+  ram_size: 778240
+  rom_size: 241664
+
+- scenarios:
+    - sample.matter.lock.debug
+    - sample.matter.light_bulb.debug
+  platforms:
+    - nrf52840dk_nrf52840
+  ram_size: 931840
```

Comment:
btw. Aren't ROM and RAM values actually swapped for all of the records? `nrf52840` has 256kB of RAM so it would never reach the threshold here

Reply:
yes, my mistake. Thanks